hw/ip/i2c/doc/_index.md

```diff
@@ -1,3 +1,223 @@
+
 ---
-title: "I2C Technical Specification" 
+title: "I2C HWIP Technical Specification"
 ---
+
+
+# Overview 
+
+This document specifies I2C hardware IP functionality.
+This module conforms to the [Comportable guideline for peripheral functionality.](https://github.com/lowRISC/stwg-base/blob/master/doc/ComportabilitySpecification.md)
+See that document for integration overview within the broader top level system.
+
+
+## Features 
+
+- Two pin clock-data parallel bidirectional external interface 
+- The initial revision only supports I2C as a Host (I2C Master).
+- Support for standard (100 kbaud), fast (400 kbaud) and fast-plus (1 Mbaud) modes
+- Bandwidth up to 1 Mbaud
+- Support for all “Mandatory” features as specified for I2C Hosts (as listed in Table 2 of the I2C specification): 
```

Comment:
- can you link to the specification (most likely https://www.nxp.com/docs/en/user-guide/UM10204.pdf)?
- Use ASCII quotation marks.

Reply:
Oof.   Sneaky google docs and those non-ascii quotes ;)  WIll do.

As for the link.  There is one there elsewhere.  I can add a second one here though.

Reply:
Added link and ran entire file through iconv -t ASCII to find non-ascii chars.